src/bluesky_stomp/messaging.py

```diff
@@ -224,7 +236,12 @@ def _send_bytes(
             )
         if correlation_id:
             headers = {**headers, CORRELATION_ID_HEADER: correlation_id}
-        self._conn.send(headers=headers, body=message, destination=destination)  # type: ignore
+
+        tracer = get_tracer("_send_bytes")
+
+        with tracer.start_as_current_span("_send_bytes"):
```

Comment:
This is not really telling us much, other than we called _send_bytes. Better would be to use the @start_as_current_span decoratoron the method so you can capture the message and destination as Span Attributes as well as the fact that the method was called